Yank/commands/script.py

```diff
@@ -38,7 +38,11 @@
                                 1 <= job_id <= n_jobs. In this case, njobs must be specified as well and YANK will
                                 run only 1/n_jobs of the experiments. This can be used to run several separate YANK
                                 executions in parallel starting from the same script.
+                                Alternatively, a zero-indexed environment variable name can be specified, in which
+                                case the jobid will be taken from this variable (0 <= environment variable < n_jobs).
```

Comment:
As I mentioned above, I don't think this will work. The index must be `1 <= jobid <= njobs` or `ExperimentBuilder` will complain.

Reply:
I changed [the code](https://github.com/choderalab/yank/pull/897/files#diff-a415ceb8df1d0ac62f46ca6dd0d5bb71R107) to add 1 after you pointed this out! I've confirmed it works on titan.

Reply:
Ah! Sorry, I didn't notice the `+1`. Feel free to merge then!